nixos/lib/make-system-tarball.sh

```diff
@@ -14,8 +13,6 @@ stripSlash() {
     if test "${res:0:1}" = /; then res=${res:1}; fi
 }
 
-touch pathlist
```

Comment:
Do you know why this was there?

Reply:
Original `nixos/lib/make-system-tarball.sh` does not have any further reference to `pathlist`. However, I have a theory that this script was written in tandem with `make-iso9660-image.sh`, since the function names and variables are very similar.